alpaca/common/websocket.py

```diff
@@ -141,7 +141,7 @@ async def _consume(self) -> None:
             else:
                 try:
                     r = await asyncio.wait_for(self._ws.recv(), 5)
-                    msgs = msgpack.unpackb(r)
+                    msgs = msgpack.unpackb(r, timestamp=3)
```

Comment:
@alessiocastrica I have just noticed that this may cause a breaking changes of raw_data response as this returns `datetime.datetime` instead of `msgpack.Timestamp`.
How about using `msg["t"].to_datetime()` in `_cast`?

ref. https://github.com/msgpack/msgpack-python/blob/2982e9ff729eae150d67ee608fdf1d01d93d8e3f/msgpack/fallback.py#L556-L557

Reply:
You're totally right, on it.